plugins/Diagnostics/Diagnostic/Diagnostic.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+/**
+ * Performs a diagnostic on the system or Piwik.
```

Comment:
> is it recommended to use a translation?

Not more than anywhere else in Piwik. 3rd party plugins sometimes don't care about translations, there's no difference here. If we do recommend to use translation in comments elsewhere in Piwik why not add it here too, but if not then we shouldn't care here more than elsewhere.